eradiate/scenes/biosphere/_discrete.py

```diff
@@ -151,7 +170,7 @@ def _leaf_cloud_orientations(n_leaves, mu, nu, rng):
     return orientations
 
 
-@ureg.wraps(ureg.m, (None, ureg.m))
+@ureg.wraps(ureg.m, (None, ureg.m), strict=False)
```

Comment:
Please keep `strict` to `True` and wrap arguments into appropriate units in calling code (see `test_leaf_cloud_radii()` for the corresponding unit test). ` strict=False` allows to pass unitless quantities and might result in hard-to-predict behaviour.